task-launcher/src/styles/components/_buttons.scss

```diff
@@ -228,4 +228,30 @@ button {
       border-color: $border-button-primary-clicked;
     }
   }
+
+  &.image-medium {
+    @extend .button-base; 
+    padding: 0; 
+    box-sizing: border-box; 
+    border-color: $bg-button-primary;
+
+    img {
+      border: none;
+      width: $response-size-large;
+      height: $response-size-large;
+      padding: 0;
+      margin: 0;
+      border-radius: $lev-spacing-s;
+      float: left;
+    }
+
+    &:hover {
+      border-color: $bg-accent_secondary;
+    }
+
+    &:active {
+      border-color: $border-button-primary-clicked;
```

Comment:
This will only change the default size and not affect the other screens. This should probably be tested across a range of screen sizes

Reply:
I tested it with as many screen sizes as I could in Chrome and none of them cause the buttons to go out of screen, but let me know if it would be better to do it a different way!

Reply:
@zwatson2001 pushed some changes to make the system a little more responsive. Please let me know if you have questions. I'll merge this tomorrow AM your time.